master/internal/webhooks/authz_basic_impl.go

```diff
@@ -0,0 +1,19 @@
+package webhooks
+
+import (
+	"github.com/determined-ai/determined/master/pkg/model"
+)
+
+// WebhookAuthZBasic is basic OSS controls.
+type WebhookAuthZBasic struct{}
+
+// CanEditWebhooks always returns true and a nil error.
+func (a *WebhookAuthZBasic) CanEditWebhooks(
+	curUser *model.User,
+) (canEditWeb bool, serverError error) {
+	return true, nil
```

Comment:
Not exactly sure but feels more OSS permission style to require admin here?

Reply:
You are correct this has been updated so only admins can manage webhooks.